compiler/rustc_middle/src/query/mod.rs

```diff
@@ -2576,6 +2576,12 @@ rustc_queries! {
         desc { "estimating codegen size of `{}`", key }
         cache_on_disk_if { true }
     }
+
+    query anon_const_kind(def_id: DefId) -> ty::AnonConstKind {
+        desc { |tcx| "looking up anon const kind of `{}`", tcx.def_path_str(def_id) }
+        cache_on_disk_if { def_id.is_local() }
```

Comment:
typically we don't cache on disk unless perf suggests that it's necessary -- esp b/c the query impl looks pretty cheap. Can you remove it, and we'll perf it if it's necessary?

Reply:
Oh do you not need to do this to get it to work in cross-crate situations? No reason to have it then you're right

Reply:
No, `cache_on_disk` is just for caching the results of the query for perf.